coverage/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+# Licensed under the Apache-2.0 license
+
+[package]
+name = "caliptra-coverage"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow.workspace = true
+serde = {workspace = true, features = ["derive"]}
+serde_json.workspace  = true
+random_name_generator = { version = "0.3.2" }
```

Comment:
This thing is adding way too many deps to Cargo.lock relative to it's value IMHO. (regex? anstyle?)

Can you use hex::encode(rand::random::<[u8; 16]>()) intsead? We already have deps on `hex` and `rand`

Reply:
Done for this PR.